taf/testlib/virtual_env.py

```diff
@@ -107,7 +109,7 @@ def __init__(self, opts=None):
         self.opts = opts
         self.settings = self._get_settings(self.opts.env)
         self.tempest_path = self.opts.tempest_path
-        self.reuse_venv = self.opts.reuse_venv
+        self.reuse_venv = eval(self.opts.reuse_venv)
```

Comment:
Please avoid using eval.  If the value is JSON format, then use `json.loads()`, otherwise use `ast.literal_eval()`.  https://docs.python.org/3/library/ast.html#ast.literal_eval